src/main/java/com/fpeng2288/requestservice/RequestserviceApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.fpeng2288.requestservice;
+
```

Comment:
Maybe we should use organization name instead of developer name in the package name. 
Volunteer service is using this
![image](https://github.com/saayam-for-all/request/assets/43979219/aec81995-0c3b-433c-a873-bc842584aa0e)

It would be good if we discuss as a team in daily stand-up and keep consistency across all code.

Reply:
I couldn't agree more, replace the current one when it is specifically decided whether to use 'sfa' or 'saayam'.

Reply:
Hi @FPeng2288 ,
Looks like we have agreed to use package name as "com.sfa.XXX" on WhatsApp. Will you be able to update the package names accordingly and update us? Please let me know if you need any help with updating the names :).
Thanks

Reply:
I'll upload a new PR containing the changed package name as soon as possible.